karma.conf.js

```diff
@@ -16,6 +16,7 @@ module.exports = (config) => {
       'node_modules/font-awesome/css/font-awesome.css',
       'build/hglib.css',
       'test/**/*.+(js|jsx)',
+      //'test/RuleTests.js',
```

Comment:
```suggestion
      // 'test/RuleTests.js',
```

This is causing troubles with Travis because of a lint issue :)